lighthouse-core/audits/byte-efficiency/uses-responsive-images.js

```diff
@@ -80,8 +81,7 @@ class UsesResponsiveImages extends ByteEfficiencyAudit {
 
   /**
    * @param {!Artifacts} artifacts
-   * @return {{results: !Array<Object>, tableHeadings: Object,
-   *     passes: boolean=, debugString: string=}}
+   * @return {{results: !Array<Object>, headings: !Audit.Headings, passes: boolean=, debugString: string=}}
```

Comment:
`boolean=` would be `(boolean|undefined)`, but doesn't look like it can be undefined?
`string=` would be `(string|undefined)`

(`=` only works for `@params`...technically it means optional param, which just so happens to manifest as being `undefined`)